pom.xml

```diff
@@ -9,15 +9,15 @@
 
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <BukkitVersion>1.8-R0.1-SNAPSHOT</BukkitVersion>
+        <SpigotVersion>1.12.2-R0.1-SNAPSHOT</SpigotVersion>
         <VaultVersion>1.3.01</VaultVersion>
     </properties>
 
     <dependencies>
         <dependency>
-            <groupId>org.bukkit</groupId>
-            <artifactId>craftbukkit</artifactId>
-            <version>${BukkitVersion}</version>
+            <groupId>org.spigotmc</groupId>
```

Comment:
Please revert this. Plugin builds fine against spigot, but we do not rely on anything spigot-specific. This artifact should be org.bukkit.craftbukkit, because there are multiple implementations of bukkit api (spigot being one of them).